dataactbroker/scripts/update_detached_procurement.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from dataactcore.logging import configure_logging
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.models.jobModels import Submission # noqa
+from dataactcore.models.userModel import User # noqa
```

Comment:
If we aren't importing any other models (now that we're just doing raw sql here), does it still throw errors without Submission and User? It might very well be that it does, but if it doesn't then we should take these imports out.

Reply:
Good point. seems like it is not necessary because we use raw sql instead of sqlalchemy there is no need for the models